### DISCUSSION TITLE
Update-DbaBuildReference: Supply Proxy via Parameter

### DISCUSSION THREAD

**Discussion by fm-knopfler** [dataplat/dbatools#9172] [Ideas] [answered]
When the connection to the online source fails, Update-DbaBuildReference will try to use the default proxy. In my usecase, beeing able to supply a proxy via parameter would be very useful.

Would this be useful to you, too? If there's demand, I can file a PR.

```powershell
Update-DbaBuildReference -Proxy $Proxy -ProxyCredential $ProxyCred
```

**Comment by niphlod** [comment, marked as answer]
hi @fm-knopfler , as long as its optional and gets passed down to Invoke-TlsWebRequest, and also those are optional (because Invoke-TlsWebRequest is used in another few functions) I'd say go for it.
